src/ansys/fluent/core/solver/flobject.py

```diff
@@ -292,6 +292,13 @@ class Real(SettingsBase[RealType], Numerical):
     expression values.
     """
 
+    def quantity_map(self) -> Dict[str, float]:
+        """Get the units-quantity of the object."""
+        val = self.get_attr("units-quantity", str)
+        if isinstance(val, str):
+            val = ' '.join(word.capitalize() for word in val.lstrip("'").split("-"))
```

Comment:
@hpohekar we won't merge the version with capitalize. We are using a different set of names from the settings API. Either we conform to those names (either directly or via some adapter) or the settings API provides the names in our (Ansys) format.